package.json

```diff
@@ -193,7 +193,7 @@
   },
   "dependencies": {
     "@babel/runtime": "7.20.6",
-    "@scaleway/ui": "0.212.7",
+    "@scaleway/ui": "0.212.8",
```

Comment:
Can we update to the 0.212.9 at the same time ?

Reply:
mmmh sure what changed in 212.9?

Reply:
@matthprost if we need to change anything let me know